classes/log/store.php

```diff
@@ -96,22 +97,22 @@ public function process_events(array $events) {
         // Emits events to other APIs.
         foreach ($events as $event) {
             $event = (array) $event;
-            // $this->error_log('');
-            // $this->error_log_value('event', $event);
+            $this->error_log('');
+            $this->error_log_value('event', $event);
             $moodleevent = $moodlecontroller->createEvent($event);
             if (is_null($moodleevent)) {
                 continue;
             }
-
-            // $this->error_log_value('moodleevent', $moodleevent);
-            $translatorevent = $translatorcontroller->createEvent($moodleevent);
-            if (is_null($translatorevent)) {
+            $this->error_log_value('moodleevent', $moodleevent);
+            $translatorevents = $translatorcontroller->createEvents($moodleevent);
+            if (is_null($translatorevents)) {
```

Comment:
My recent change to the controller in the PR for multiple translator events means that this check will no longer be necessary because it just returns an empty array rather than `null`.